pyperformance/venv.py

```diff
@@ -137,6 +137,12 @@ def _get_envvars(inherit=None, osname=None):
         copy_env.extend(inherit)
 
     env = {}
+    # Recent setuptools versions have a bug that prevents being
+    # used to compile from a source build virtual environment.
+    # [BUG](https://github.com/pypa/setuptools/issues/3325)
+    # The stdlib distutils does not have this problem.
```

Comment:
What happens when the stdlib distutils goes away?

Reply:
Presumably the bug would have to be fixed in setuptools?